shell/platform/windows/public/flutter_windows.h

```diff
@@ -133,7 +133,7 @@ FLUTTER_EXPORT bool FlutterDesktopViewControllerHandleTopLevelWindowProc(
 // The caller owns the returned reference, and is responsible for calling
 // FlutterDesktopEngineDestroy.
 FLUTTER_EXPORT FlutterDesktopEngineRef FlutterDesktopEngineCreate(
-    const FlutterDesktopEngineProperties& engine_properties);
+    const FlutterDesktopEngineProperties* engine_properties);
```

Comment:
Optional: maybe document that it doesn't need to live beyond the call, since that's no longer apparent from the type.

Reply:
Done. Thanks for the suggestion!